### PR TITLE
Cleanup Borzoi

### PR DESCRIPTION
**Pull request by marenz2569**
Implement the packet to and from json for borzoi integration. Closes https://github.com/tlm-solutions/tetra-decoder/issues/21

### CODE REVIEW

File: include/nlohmann/std_unique_ptr_logical_link_control_packet.hpp [hunk 1]
```diff
@@ -26,25 +34,25 @@ template <> struct adl_serializer<std::unique_ptr<LogicalLinkControlPacket>> {
             if (auto* cmce = dynamic_cast<CircuitModeControlEntityPacket*>(mle)) {
                 if (auto* sds = dynamic_cast<ShortDataServicePacket*>(mle)) {
                     // Emit ShortDataServicePacket packet to json
-                    j["key"] = "ShortDataServicePacket";
+                    j["key"] = PacketType::kShortDataServicePacket;
```
Comment (tanneberger) [inline include/nlohmann/std_unique_ptr_logical_link_control_packet.hpp:37]: nice!

File: include/l3/short_data_service_packet.hpp [hunk 2]
```diff
@@ -14,40 +14,54 @@
 #include <optional>
 
 struct ShortLocationReport {
-    unsigned _BitInt(2) time_elapsed_;
-    double longitude_;
-    double latitude_;
+    unsigned _BitInt(2) time_elapsed_{};
+    double longitude_{};
```
Comment (tanneberger) [inline include/l3/short_data_service_packet.hpp:18]: why not initialize with a proper value?

Comment (marenz2569) [reply include/l3/short_data_service_packet.hpp:18]: default initializer to 0

File: include/utils/ostream_std_unique_ptr_logical_link_control_packet.hpp [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+inline auto operator<<(std::ostream& stream, const std::unique_ptr<LogicalLinkControlPacket>& packet) -> std::ostream& {
+    /// process the parsed packet
+    auto* cplane_signalling = dynamic_cast<UpperMacCPlaneSignallingPacket*>(packet.get());
+    auto* llc = dynamic_cast<LogicalLinkControlPacket*>(cplane_signalling);
+
+    stream << *cplane_signalling;
+    stream << *llc;
+    if (auto* mle = dynamic_cast<MobileLinkEntityPacket*>(llc)) {
+        stream << *mle;
+        if (auto* cmce = dynamic_cast<CircuitModeControlEntityPacket*>(llc)) {
```
[the remark this answers is not in the document]
Comment (marenz2569) [reply include/utils/ostream_std_unique_ptr_logical_link_control_packet.hpp:24]: this design choice is not part of this pr

**Pull request merged**